Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM golang:1.10-stretch
+# Install software
+RUN apt-get update
+RUN apt-get install -y bind9 python3 python3-pip
```

Comment:
To optimize the size of the docker container (and speed up CI as a result) we should clean up after using apt-get. 

For an example, take a look at another one of our containers:

https://github.com/ansible/cloudstack-test-container/blob/36560615b4a4f7f7e59bbc69c87b02ddccedc255/Dockerfile#L8-L28

This would be something good to update for the next version of the container.

Reply:
Good idea. PR will (hopefully) follow soon. :)